f8a_worker/dispatcher/nodes.yml

```diff
@@ -280,12 +280,18 @@
       max_retry: 0
       queue: '{DEPLOYMENT_PREFIX}_{WORKER_ADMINISTRATION_REGION}_KeywordsSummaryTask_tagging_v0'
       storage: 'S3KeywordsSummary'
-    - name: 'dependency_tree'
+    - name: 'GithubDependencyTreeTask'
       classname: 'GithubDependencyTreeTask'
       import: 'f8a_worker.workers'
       max_retry: 0
       queue: '{DEPLOYMENT_PREFIX}_{WORKER_ADMINISTRATION_REGION}_GithubDependencyTreeTask_v0'
       storage: 'BayesianPostgres'
+    - name: 'unknown_deps_fetcher'
```

Comment:
Uniform naming convention will definitely help

Reply:
@miteshvp Similar convention was followed in erstwhile stackAnalysisV2Flow as well, but I will surely make this uniform in an upcoming PR after the complete flow is implemented.